ConsoleToGo/ConsoleToGo.vcxproj

```diff
@@ -108,6 +109,9 @@
       <ProgramDatabaseFile />
       <GenerateDebugInformation>false</GenerateDebugInformation>
     </Link>
+    <PreBuildEvent>

```

Comment:
Again, not needed for this project as the version is that of the deployed app, not of Dolphin